doc/source/installation/apt_repo.rst

```diff
@@ -34,34 +34,34 @@ Installing |Percona Server| from Percona ``apt`` repository
 
    .. code-block:: bash
 
-      $ sudo apt-get install gnupg2
+      sudo apt install gnupg2
```

Comment:
Just curious: you removed the dollar prompt from some command line examples, while you kept it on others.

What's the general rule here? I usually prefer using the `$` to indicate a command that I enter as a regular user, while the `#` symbol indicates commands to be entered as the `root` user. SQL statements should be prefixed by e.g. `mysql>` when using the MySQL CLI.